src/datasources/push-notifications-api/entities/firebase-oauth2-token.entity.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from 'zod';
```

Comment:
Minor note: This change is absolutely fine as is. Initially, I was a bit confused since the change didn’t seem directly related to the Raw type. However, I understand that it falls under the same domain due to its connection with validation.

Reply:
I agree that it is somewhat out of place here - I forgot to move it to it's own PR! Would you like me to create a separate one?

Reply:
No. As I said it's totally fine now. But for future it would be best to do so. I'll resolve the thread